bitnami/memcached/templates/statefulset.yaml

```diff
@@ -0,0 +1,119 @@
+{{- if eq .Values.architecture "high-availability" }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "memcached.fullname" . }}
+  labels: {{- include "memcached.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels: {{- include "memcached.matchLabels" . | nindent 6 }}
+  replicas: {{ .Values.replicaCount }}
+  serviceName: {{ template "memcached.fullname" . }}-headless
+  template:
+    metadata:
+      labels: {{- include "memcached.labels" . | nindent 8 }}
+      {{- if or .Values.podAnnotations (and .Values.metrics.enabled .Values.metrics.podAnnotations) }}
+      annotations:
+        {{- if .Values.podAnnotations }}
+        {{- include "memcached.tplValue" (dict "value" .Values.podAnnotations "context" $) | nindent 8 }}
+        {{- end }}
+        {{- if and .Values.metrics.enabled .Values.metrics.podAnnotations }}
+        {{- include "memcached.tplValue" (dict "value" .Values.metrics.podAnnotations "context" $) | nindent 8 }}
+        {{- end }}
+      {{- end }}
+    spec:
+{{- include "memcached.imagePullSecrets" . | indent 6 }}
+      {{- if .Values.affinity }}
+      affinity: {{- include "memcached.tplValue" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- include "memcached.tplValue" (dict "value" .Values.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- include "memcached.tplValue" (dict "value" .Values.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.securityContext.enabled }}
+      securityContext:
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+      {{- end }}
+      containers:
+        - name: memcached
+          image: {{ template "memcached.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          command:
+{{ toYaml .Values.command | indent 12 }}
```

Comment:
Same suggestion about using a single line:
```yaml
          command: {{- toYaml .Values.command | nindent 12 }}
```

Reply:
I missed the whole command vs args thing. I think we're back to where it should be and properly using entrypoint (e.g. default command) now. Might be more to tweak.